ska_conda/pkg_defs/ska3-template/bin/flt_envs

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python -E
+#!PREFIXPYTHON -E
```

Comment:
Could use /usr/bin/python here as well, I just figured there was less of a reason to do so if we are guaranteed to have a Python in our conda environment by the recipe requirements.  So the build.sh for this recipe just substitutes in the prefix.  At least that is the plan.

Reply:
I rebased these changes away for now and put them in a new PR.  Will come back to this.